elliottlib/bzutil.py

```diff
@@ -606,8 +607,11 @@ def remove_bugs(self, advisory_obj, bugids: List, noop=False):
         if noop:
             print(f"Would've removed bugs: {bugids}")
             return
-        advisory_id = advisory_obj.errata_id
-        return errata.remove_multi_jira_issues(advisory_id, bugids)
+        try:
+            advisory_obj.removeJIRAIssues(bugids)
+            advisory_obj.commit()
+        except ErrataException as e:
+            raise exceptions.ElliottFatalError(getattr(e, 'message', repr(e)))
```

Comment:
I didn't see a strong reason to capture and reraise a different exception here. This will lose the stack trace info.